File: src/main/java/com/gitlab/daring/image/template/MatchMethod.java
package com.gitlab.daring.image.template;

public enum MatchMethod {

    SQDIFF, SQDIFF_NORMED, CCORR, CCORR_NORMED, CCOEFF, COEFF_NORMED;

    public boolean isMinBest() {
        return this == SQDIFF || this == SQDIFF_NORMED;
    }

}

File: src/main/kotlin/com/gitlab/daring/image/command/template/MatchMethod.kt
package com.gitlab.daring.image.command.template

internal enum class MatchMethod {

    SQDIFF, SQDIFF_NORMED, CCORR, CCORR_NORMED, CCOEFF, COEFF_NORMED;

    val isMinBest: Boolean
        get() = this == SQDIFF || this == SQDIFF_NORMED

}